test/tests/misc/debug_pid_aux/debug_pid_aux.i

```diff
@@ -0,0 +1,28 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 8
+  ny = 8
+  [Partitioner]
+    type = GridPartitioner
+    nx = 2
+    ny = 2
+    nz = 1
```

Comment:
Just let you know parameters ` nx ny nz` not  necessary anymore. However your use is still definitely right